ctapipe_io_lst/pointing.py

```diff
@@ -34,42 +40,38 @@ def __init__(self, subarray, config=None, parent=None, **kwargs):
         self.drive_report = {}
         self.interp_az = {}
         self.interp_alt = {}
-        self.interp_ra = {}
-        self.interp_dec = {}
 
     @staticmethod
     def _read_drive_report(path):
         """
         Read a drive report into an astropy table
 
-        Parameters:
-        -----------
-        str: drive report file
-
-        Returns:
-        data:`~astropy.table.Table`
-             A table of drive reports
+        Parameters
+        ----------
+        path : str or Path
+            drive report file
 
+        Returns
+        -------
+        data : `~astropy.table.Table`
+            A table of drive reports
         """
         path = Path(path)
         Provenance().add_input_file(str(path), 'drive positioning')
+
         data = Table.read(
             path, format='ascii', delimiter=' ',
             header_start=None,
             data_start=0,
-            # the Az, Zd, Ra, Dec columns are just dummies, the file
-            # always contains "Az", "Zd" (or "El" for older files), "Ra", "Dec"
-            names=[
-                'weekday', 'month', 'day', 'time', 'year', 'unix_time',
-                'Az', 'azimuth_avg', 'azimuth_min', 'azimuth_max', 'azimuth_std',
-                'Zd', 'zenith_avg', 'zenith_min', 'zenith_max', 'zenith_std',
-                'Ra', 'target_ra', 'Dec', 'target_dec',
-            ],
-            exclude_names=['weekday', 'month', 'day', 'time', 'year', 'Az', 'Zd', 'Ra', 'Dec'],
+            names=["unix_time", "azimuth", "zenith"],
         )
 
         # check for bending model corrections
-        bending_path = path.with_name(path.name.replace('drive', 'bendingmodelcorrection'))
+        if "DrivePosition" not in path.name:
```

Comment:
A bit misleading at first glance since you should, in principle, read files named DrivePosition...